python/lsst/cp/verify/verifyBias.py

```diff
@@ -68,18 +69,25 @@ def verify(self, exposure, statisticsDict):
         """
         detector = exposure.getDetector()
         ampStats = statisticsDict['AMP']
+        metadataStats = statisticsDict['METADATA']
 
         verifyStats = {}
         success = True
         for ampName, stats in ampStats.items():
             verify = {}
-
+            readNoiseMatch = True
             # DMTN-101 Test 4.2: Mean is 0.0 within noise.
             verify['MEAN'] = bool(np.abs(stats['MEAN']) < stats['NOISE'])
 
             # DMTN-101 Test 4.3: Clipped mean matches readNoise.
-            amp = detector[ampName]
-            verify['NOISE'] = bool(np.abs(stats['NOISE'] - amp.getReadNoise())/amp.getReadNoise() <= 0.05)
+            readNoise = detector[ampName].getReadNoise()
+            if 'RESIDUAL STDEV' in metadataStats and ampName in metadataStats['RESIDUAL STDEV']:
+                overscanReadNoise = metadataStats['RESIDUAL STDEV'][ampName]
```

Comment:
OK, here you assign the per amp `RESIDUAL STDEV`  to a variable with a more descriptive name, but maybe a comment would still be good.